src/libs/costmap_2d/src/costmap_2d_node.cpp

```diff
@@ -35,18 +35,21 @@
  * Author: Eitan Marder-Eppstein
  *         David V. Lu!!
  *********************************************************************/
-#include <ros/ros.h>
+#include <rclcpp/rclcpp.hpp>
 #include <costmap_2d/costmap_2d_ros.h>
 #include <tf2_ros/transform_listener.h>
 
-int main(int argc, char** argv)
+int main(int argc, char ** argv)
 {
-  ros::init(argc, argv, "costmap_node");
-  tf2_ros::Buffer buffer(ros::Duration(10));
+  rclcpp::init(argc, argv);
+
+  auto node = rclcpp::Node::make_shared("costmap_2d");
+  std::string name = (argc == 1) ? "costmap" : std::string(argv[1]);
```

Comment:
I don't think we should do this. There will likely be other arguments on the command line. Just leave the code to work as it used to.